anymarket-sdk-order/src/main/java/br/com/anymarket/sdk/order/dto/TrackingResource.java

```diff
@@ -103,18 +109,36 @@ public void setDeliveredDate(Date deliveredDate) {
         this.deliveredDate = deliveredDate;
     }
 
+    public Long getDeliveryMethodId() {
+        return deliveryMethodId;
+    }
+
+    public void setDeliveryMethodId(Long deliveryMethodId) {
+        this.deliveryMethodId = deliveryMethodId;
+    }
+
+    public String getDeliveryMethodName() {
+        return deliveryMethodName;
+    }
+
+    public void setDeliveryMethodName(String deliveryMethodName) {
+        this.deliveryMethodName = deliveryMethodName;
+    }
+
     @Override
     public String toString() {
         return MoreObjects.toStringHelper(this)
-            .add("url", url)
-            .add("number", number)
-            .add("carrier", carrier)
-            .add("carrierDocument", carrierDocument)
-            .add("date", date)
-            .add("estimateDate", estimateDate)
-            .add("shippedDate", shippedDate)
-            .add("deliveredDate", deliveredDate)
-            .toString();
+                .add("url", url)
+                .add("number", number)
+                .add("carrier", carrier)
+                .add("carrierDocument", carrierDocument)
+                .add("date", date)
+                .add("estimateDate", estimateDate)
+                .add("shippedDate", shippedDate)
+                .add("deliveredDate", deliveredDate)
+                .add("deliveryMethodId", deliveryMethodId)
+                .add("deliveryMethodName", deliveryMethodName)
+                .toString();
```

Comment:
```suggestion
              .add("url", url)
              .add("number", number)
              .add("carrier", carrier)
              .add("carrierDocument", carrierDocument)
              .add("date", date)
              .add("estimateDate", estimateDate)
              .add("shippedDate", shippedDate)
              .add("deliveredDate", deliveredDate)
              .add("deliveryMethodId", deliveryMethodId)
              .add("deliveryMethodName", deliveryMethodName)
              .toString();
```

Só remover as linhas extras adicionadas.

Reply:
@williamwillers

Reply:
Linhas adicionais removidas